src/main/java/org/grnet/pidmr/service/MailerService.java

```diff
@@ -87,25 +120,50 @@ public void sendMails(RoleChangeRequest request, MailType type, List<String> mai
         templateParams.put("pidmr", uiBaseUrl);
         templateParams.put("title", apiName.toUpperCase());
 
+        templateParams.put("id", String.valueOf(emailContext.getRequestStatus()));
```

Comment:
templateParams.put("id", String.valueOf(emailContext.getRequestStatus())); 
here should be a emailContext.getId() instead of emailContext.getRequestStatus()